examples/with-viem/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+
+import { createApiKeyAccount } from "@turnkey/viem";
+import { createWalletClient, http } from "viem";
+import { sepolia } from "viem/chains";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+async function main() {
+  const turnkeyAccount = await createApiKeyAccount({
+    apiPublicKey: process.env.API_PUBLIC_KEY!,
+    apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    baseUrl: process.env.BASE_URL!,
+    organizationId: process.env.ORGANIZATION_ID!,
+    privateKeyId: process.env.PRIVATE_KEY_ID!,
+  });
+
+  const client = createWalletClient({
+    account: turnkeyAccount,
+    chain: sepolia,
+    transport: http(
+      `https://sepolia.infura.io/v3/${process.env.INFURA_API_KEY!}`
```

Comment:
should we include the default community API key somewhere? 
`84842078b09946638c03157f83405213`

Reply:
Oh yeah. I can include it in the example env file 👍